examples/sample_plugin/otio_counter/__init__.py

```diff
@@ -62,6 +63,6 @@ def plugin_manifest():
     # XXX: note, this doesn't get called.  For an example of this working,
     #      see the mockplugin unit test.
 
-    return manifest.manifest_from_string(
-        pkg_resources.resource_string(__name__, 'plugin_manifest.json')
-    )
+    with importlib.resources.path(__name__, "plugin_manifest.json") as p, \
+            open(p, 'rb') as f:
+        return manifest.manifest_from_string(f.read())
```

Comment:
```suggestion
    with (
            importlib.resources.path(__name__, "plugin_manifest.json") as p,
            open(p, 'rb') as f
    ):
        return manifest.manifest_from_string(f.read())
```
Style preference for me, consider this optional - I'm not a fan of "\" for line continuation, would rather use ()